pkg/controller/jobframework/reconciler.go

```diff
@@ -236,6 +255,22 @@ func (r *JobReconciler) ReconcileGenericJob(ctx context.Context, req ctrl.Reques
 	return ctrl.Result{}, nil
 }
 
+// isParentJobManaged checks whether the parent job is managed by kueue.
+func (r *JobReconciler) isParentJobManaged(ctx context.Context, jobObj client.Object, namespace string) (bool, error) {
+	owner := metav1.GetControllerOf(jobObj)
+	if owner == nil {
+		return false, errChildJobOwnerNotFound
+	}
+	parentJob := KnownWorkloadOwnerObject(owner)
+	if parentJob == nil {
+		return false, fmt.Errorf("workload owner %s: %w", owner.String(), errUnknownWorkloadOwner)
```

Comment:
I think this should work?

```suggestion
		return false, fmt.Errorf("workload owner %s: %w", owner, errUnknownWorkloadOwner)
```

Reply:
done.